### DISCUSSION TITLE
Can json schema support any language except json/yaml/toml?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [json-schema-org/community#428] [Q&A] [answered]
<https://www.schemastore.org/json/> support json/yaml/toml. Can it support dosini (Linux desktop entry, systemd unint, etc)?

**Comment by gregsdennis** [comment, marked as answer]
Technically any syntax that can represent the JSON data model (objects, arrays, numbers, strings, true, false, & null) can be processed by JSON Schema.

The trick is translating whatever syntax you have into that data model.

---

Secondly, Schema Store isn't ours.  We control JSON Schema, the spec.  Schema Store is a separate website that manages schemas is various formats.
